### PR TITLE
Feature/initial position in stream option

### PR DESCRIPTION
**Pull request by aloisbarreras**

### CODE REVIEW

File: lib/logstash/inputs/kinesis.rb [hunk 1]
```diff
@@ -53,6 +53,9 @@ class LogStash::Inputs::Kinesis < LogStash::Inputs::Base
   # Select AWS profile for input
   config :profile, :validate => :string
 
+  # Select initial_position_in_stream. Accepts TRIM_HORIZON or LATEST
+  config :initial_position_in_stream, :validate => :string, :default => "TRIM_HORIZON"
```
Comment (codekitchen) [inline lib/logstash/inputs/kinesis.rb]: let's change `:validate => :string` to `:validate => ["TRIM_HORIZON", "LATEST"]`, this way they can't enter some other random string and end up still getting LATEST due to the conditional below. See the `:metrics` config var above for an example.

**Comment by aloisbarreras** [comment]
@codekitchen awesome! I don't know what your release cadence is, but it would be great if we could get this released soon.

**Comment by RyanPnayR** [comment]
Would be great if this could get merged in asap.

**Comment by codekitchen** [comment]
Looks like the CI build eventually ran. Looks good, thanks again.

**Pull request merged**